coderd/database/queries/insights.sql

```diff
@@ -302,7 +324,17 @@ SELECT
 	ai.slug,
 	ai.display_name,
 	ai.icon,
-	(SUM(ai.usage_mins) * 60)::bigint AS usage_seconds
+	(SUM(ai.usage_mins) * 60)::bigint AS usage_seconds,
+	COALESCE((
+		SELECT
+			COUNT(*)
```

Comment:
It is just about cardinality of app usage, right?

Reply:
Yes, we are only interested in rows of this table, not of it's content.